pwa/package.json

```diff
@@ -75,8 +75,6 @@
     "@types/react-dom": "18.3.1",
     "@typescript-eslint/eslint-plugin": "8.23.0",
     "@typescript-eslint/parser": "8.23.0",
-    "@vite-pwa/assets-generator": "0.2.6",
```

Comment:
Do we need to remove the assets generator? if so we should also remove the `generate-pwa-assets` script above.